gpflow/optimizers/scipy.py

```diff
@@ -74,12 +76,12 @@ def minimize(
         )
 
     @classmethod
-    def initial_parameters(cls, variables: Variables) -> tf.Tensor:
+    def initial_parameters(cls, variables: Iterable[tf.Variable]) -> tf.Tensor:
         return cls.pack_tensors(variables)
 
     @classmethod
     def eval_func(
-        cls, closure: LossClosure, variables: Variables, compile: bool = True
+        cls, closure: LossClosure, variables: Tuple[tf.Variable], compile: bool = True
```

Comment:
```suggestion
        cls, closure: LossClosure, variables: Tuple[tf.Variable, ...], compile: bool = True
```
I believe `Tuple[tf.Variable]` can only be `(tf.Variable(0.),)`

Reply:
and for the others